### PR TITLE
edit greenpeace

### PR DESCRIPTION
**Pull request by yeah800**
*Please edit everything below that's UPPERCASE and starts with a "📝"*:

**What change is this pull request introducing?**
📝CHANGES

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -103,16 +103,36 @@ exports.data = [
             label: "greenpeace",
         },
         description: {
-            content: "Greenpeace is another big player. They also do work on a big scale as you can see on the Income they have to put them into their projects. Greenpeace has a lot of Actions going on.",
-            updatedBy: "Andre", updatedAt: "2019-08-18"
+            content: "Greenpeace is another big player, working in indipendent national and regional organizations. They do work on a big scale as you can see on the income and the variety of actions.",
```
Comment (reasn) [inline data.js]: independent

File: data.js [hunk 2]
```diff
@@ -103,16 +103,36 @@ exports.data = [
...
             updatedBy: "Andre", updatedAt: "2019-08-06"
         },
         kpis: {
-            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions",
-            updatedBy: "Andre", updatedAt: "2019-08-06",
+            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions \n- Website visitors",
+            updatedBy: "Andre", updatedAt: "2020-01-14",
+        },
+        actions: {
+            content: "Information again is very important: For instance Greenpeace developed a (Guide to sue your local government for inaction regarding the climate crisis)[https://storage.googleapis.com/planet4-international-stateless/2018/12/4fdd4d8a-peoples_guide_fnl_2.pdf]"+
```
Comment (reasn) [inline data.js]: `Information again is very important: For instance Greenpeace developed a` -> `Greenpeace does a lot of PR and help for independent action. For example they developed a`

File: data.js [hunk 3]
```diff
@@ -103,16 +103,36 @@ exports.data = [
...
+                        "\n A lot of their actions of civil disobidience attract attention of the press like at the (auto show)[https://www.thelocal.de/20190914/climate-protesters-to-disrupt-frankfurt-auto-show] in Frankfurt, Germany."+
```
Comment (reasn) [inline data.js]: `disobedience`

File: data.js [hunk 4]
```diff
@@ -103,16 +103,36 @@ exports.data = [
...
+            content: "[27 offices](https://www.greenpeace.org/international/explore/about/worldwide/) worldwide.",
+            updatedBy: "Andre", updatedAt: "2020-01-14",
+        },
+        yearlyIncome: {
+            content: "- 360M € in [2018](https://drive.google.com/file/d/1y2OzoAnRBcQh5MX2hjO_iwjhy8mQJ4vC/view#page=28)" +
+                "\n- 360M € in [2017](https://drive.google.com/file/d/1y2OzoAnRBcQh5MX2hjO_iwjhy8mQJ4vC/view#page=28)",
+            updatedBy: "Andre", updatedAt: "2020-01-14"
+        },
+        pressArticles: {
+            content: "They have a [press section](https://www.greenpeace.org/international/press-centre/) for their own releases, which the press uses all around the world.",
```
Comment (reasn) [inline data.js]: afaik their own releases don't qualify as press articles. We can add a new column `pressReleases`, not sure that's necessary, would remove it

**Review by reasn — CHANGES_REQUESTED** [review]
LGTM, minor points

File: data.js (hunk 4 above)
Comment (yeah800) [reply data.js]: we could put this under actions as it is at wwf. what do you think?

File: data.js (hunk 1 above)
Comment (yeah800) [reply data.js]: check

File: data.js (hunk 2 above)
Comment (yeah800) [reply data.js]: good wording. thx

File: data.js (hunk 3 above)
Comment (yeah800) [reply data.js]: done

File: data.js (hunk 4 above)
Comment (yeah800) [reply data.js]: we have to talk about columnes anyway but I would leave it right now because we have this inforations for several other organizations.

**Review by yeah800 — COMMENTED** [review]
got it all through. take a look

**Review by yeah800 — COMMENTED** [review]
so far done(?)

File: data.js (hunk 4 above)
Comment (reasn) [reply data.js]: I wouldn't consider it relevant for any organization. It doesn't distinguish them in a relevant way and doesn't make them more comparable. `pressArticle` refers to articles about them, not by them. Happy to discuss additional columns (as said I don't think press releases are relevant)!

File: data.js (hunk 1 above)
Comment (reasn) [reply data.js]: unchanged

File: data.js (hunk 2 above)
Comment (reasn) [reply data.js]: unchanged

File: data.js (hunk 3 above)
Comment (reasn) [reply data.js]: unchanged

**Review by reasn — CHANGES_REQUESTED** [review]
some issues were not actually changed, press releases discussion remains open

**Comment by yeah800** [comment]
sorry, didn't pushed my changes last time.
Let's talk on the phone about the press releases.

File: data.js [hunk 5]
```diff
@@ -103,16 +103,36 @@ exports.data = [
             label: "greenpeace",
         },
         description: {
-            content: "Greenpeace is another big player. They also do work on a big scale as you can see on the Income they have to put them into their projects. Greenpeace has a lot of Actions going on.",
-            updatedBy: "Andre", updatedAt: "2019-08-18"
+            content: "Greenpeace is another big player, working in independent national and regional organizations. They do work on a big scale as you can see on the income and the variety of actions.",
```
Comment (reasn) [inline data.js]: Let's remove "another", it's confusing if Greenpeace is the only suggestion or on top of the list.

Comment (reasn) [reply data.js]: Brevity: `They do work on a big scale in terms of funding, staff and variety of actions`.

File: data.js [hunk 6]
```diff
@@ -103,16 +103,36 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-22"
         },
         established: {
             content: "[1971](http://www.tiki-toki.com/timeline/entry/594418/Greenpeace/#vars!date=1942-02-09_13:52:01!)",
             updatedBy: "Andre", updatedAt: "2019-08-06"
         },
         kpis: {
-            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions",
-            updatedBy: "Andre", updatedAt: "2019-08-06",
+            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions \n- Website visitors",
```
Comment (reasn) [inline data.js]: 1. I don't think "Website visitors"  is a KPI for Greenpeace
2. casing is inconsistent (Mentions, actions)

File: data.js [hunk 7]
```diff
@@ -103,16 +103,36 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-14",
+        },
+        actions: {
+            content: "Greenpeace does a lot of PR and help for independent action. For example they developed a (Guide to sue your local government for inaction regarding the climate crisis)[https://storage.googleapis.com/planet4-international-stateless/2018/12/4fdd4d8a-peoples_guide_fnl_2.pdf]"+
+                        "\n A lot of their actions of civil disobedience attract attention of the press like at the (auto show)[https://www.thelocal.de/20190914/climate-protesters-to-disrupt-frankfurt-auto-show] in Frankfurt, Germany."+
+                        "\n And of course due their wide knowledge they act as consultant for governmental and non governmental organizations.",
```
Comment (reasn) [inline data.js]: `non-governmental`

Comment (reasn) [reply data.js]: or just shorter `governments and NGOs`

File: data.js (hunk 4 above)
Comment (reasn) [reply data.js]: Still open

**Review by reasn — CHANGES_REQUESTED** [review]
and again. Few things still open, few things added

File: data.js (hunk 6 above)
Comment (yeah800) [reply data.js]: done

File: data.js (hunk 5 above)
Comment (yeah800) [reply data.js]: changed the whole wording

File: data.js (hunk 7 above)
Comment (yeah800) [reply data.js]: check

**Comment by yeah800** [comment]
all done

**Comment by reasn** [comment]
I changed two instances of brackets (`[]()`) and added your updates to crisisfighters.org

**Pull request closed without merging**